onnx2pytorch/operations/expand.py

```diff
@@ -0,0 +1,10 @@
+import torch
+from torch import nn
+
+
+class Expand(nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, input: torch.Tensor, shape: torch.Tensor):
+        return input * torch.ones(torch.Size(shape))
```

Comment:
Any reason not to use `torch.Tensor.expand` ?
https://pytorch.org/docs/master/generated/torch.Tensor.expand.html
Memory-wise seems to be more efficient since for many elements it points to the same memory location.

Reply:
Unfortunately `torch.Tensor.expand` is more restrictive than both ONNX Expand and torch's element-wise broadcasting. For example, in the ONNX docs, an example involves expanding a tensor of shape (3, 1) with shape (2, 1, 6), producing a result of shape (2, 3, 6). Yet `torch.zeros(3, 1).expand((2, 1, 6))` gives an error (`RuntimeError: The expanded size of the tensor (1) must match the existing size (3) at non-singleton dimension 1.`). Meanwhile `torch.zeros(3, 1) * torch.ones((2, 1, 6))` gives the correct result.

Reply:
Ok. Then just add some tests for the new ops please.

Reply:
@Talmaj - I've added tests, and also just now switched the PR to the develop branch.